### ISSUE TITLE
EnvO biome hierarchy is confusing and inconsistently axiomatized 

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#985]
CC @aclum @cmungall 

Alicia, Hugh, other NEON people and I are all having trouble asserting MIxS triad values to the NEON samples. I mostly attribute that to missing terms or missing relations in EnvO. I would like to put together a few examples and raise an issue with EnvO. Maybe it's better if we make a branch and PR?

**Comment by turbomam** [comment]
These challenges appear to hold for both the soil sample and the surface and benthic water samples

**Comment by turbomam** [comment]
For now, we are leaning towards making all `env_broad_scale` terms "Terrestrial biome" and all env_medium annotations `soil`

**Comment by turbomam** [comment]
For `env_local_scale` I would like to use the terms that EnvO considers equivalent to the NLCD terms asserted by NEON.

**There are all "area of..." terms. I understand that means getting into shadow classes, but I don't think there are consistent axioms from the areas to something else. I will double check.**

**Comment by turbomam** [comment]
The missing terms and inconsistent axioms may mean that users searching though the NMDC data portal may not retrieve all Biosamples that have the qualities the expect.

Like searching on `env_broad_scale` of "Terrestrial biome" may not retrieve  "subalpine biome" samples

**Comment by turbomam** [comment]
<img width="484" alt="Screen Shot 2023-06-12 at 1 07 28 PM" src="https://github.com/microbiomedata/nmdc-schema/assets/25185273/69b49384-0466-4214-bd00-0b90632c102c">

**Comment by turbomam** [comment]
Add a USDA soil classification in addition to FAO

**Comment by cmungall** [comment]
> Like searching on env_broad_scale of "Terrestrial biome" may not retrieve "subalpine biome" samples

I suggest making an ENVO issue

**Comment by aclum** [comment]
@turbomam  @cmungall  There was some work over the summer with envo, did this get resolved?

**Comment by turbomam** [comment]
Yes, [terrestrial biome](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000446) and many other classes with "biome" in the label are classified as a biome by inference now.

**Comment by turbomam** [comment]
I think there's [still work do do on EnvO biome modeling](https://github.com/EnvironmentOntology/envo/issues?q=is%3Aissue+is%3Aopen+biome), but that may not all effect us

For example, a variety of classes are asserted to be subClasses of biome but should probably be inferred.
